packages/paste-website/src/pages/foundations/content/index.mdx

```diff
@@ -39,58 +39,35 @@ export const getStaticProps = async () => {
 
 <content>
 
-**Use the guidance in the way that makes most sense to you.**
-These content pages should help you to feel more confident in writing concise, clear, and consistent product content.
 
-You might read [Voice and tone](/foundations/content/voice-and-tone) once for foundational understanding, while you might bookmark [Product style guide](/foundations/content/product-style-guide) for ease of reference.
+### [Content checklist](/foundations/content/content-checklist)
```

Comment:
All these sections should use H2s to follow the heading hierarchy order. If you feel like the size of H2s is overkill for these sections, you can use the Heading component. E.g.,:

```
<Heading as="h2" variant="heading30">
  <Anchor href="/foundations/content/content-checklist">
    Content checklist
  </Anchor>
</Heading>
```

And add the Heading component import to the top of each page:

`import {Heading} from '@twilio-paste/core/heading';`